packages/itwinui-css/src/progress-indicator/overlay.scss

```diff
@@ -15,17 +15,10 @@
   pointer-events: none;
 
   // Blur fallback
-  background-color: rgba(var(--iui-color-background-1-rgb), var(--iui-opacity-2));
-
-  // With blur
-  @supports (backdrop-filter: $iui-blur-filter) {
-    background-color: rgba(var(--iui-color-background-1-rgb), var(--iui-opacity-3));
-    backdrop-filter: $iui-blur-filter;
-  }
+  @include iui-blur($opacity: 3);
```

Comment:
This needs to be looked into:
![Screen Shot 2022-08-12 at 11 55 45 AM](https://user-images.githubusercontent.com/849817/184395224-2bf7ee53-e5f2-4bc6-9a55-9f502686e344.png)

Reply:
Fixed by adding a `-background-1-hsl` variable.